shell/config/router/navigation-guards/load-initial-settings.js

```diff
@@ -1,13 +1,15 @@
 import { fetchInitialSettings } from '@shell/utils/settings';
 
 export function install(router, context) {
-  router.beforeEach((from, to, next) => loadInitialSettings(from, to, next));
+  router.beforeEach((to, from, next) => loadInitialSettings(to, from, next, context));
 }
 
-export async function loadInitialSettings(from, to, next) {
+export async function loadInitialSettings(to, from, next, { store }) {
   try {
-    await fetchInitialSettings();
-  } catch (ex) {}
+    await fetchInitialSettings(store);
+  } catch (e) {
+    console.error('Failed fetching initial settings', e); // eslint-disable-line no-console
```

Comment:
I left this blank because it was a part of the original code but it I accidentally changed the original code and this prevented me from noticing since the error wasn't emitted anywhere.